Riot/Modules/CreateRoom/CreateRoomCoordinator.swift

```diff
@@ -37,23 +39,50 @@ final class CreateRoomCoordinator: CreateRoomCoordinatorType {
     
     // MARK: - Setup
     
-    init(session: MXSession) {
+    init(session: MXSession, parentSpace: MXSpace?) {
```

Comment:
You can use a `CreateRoomCoordinatorParameter` if you want to avoid refactor Coordinator init and BridgePresenter (if ObjC compatibility). You can see an example with the `ReauthenticationCoordinatorParameters`.